ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/DefaultApiSecurityContext.java

```diff
@@ -38,7 +38,7 @@ public static DefaultApiSecurityContext create(final RoutingContext routingConte
     }
     final ApiUser apiUser = (ApiUser) user;
 
-    String authToken = null;
+    String authToken = routingContext.request().getHeader("Authorization");
```

Comment:
Nice, I think this is equivalent to the default before these changes:
https://github.com/confluentinc/ksql/commit/b6fc8d91152db35c2f8067f4db44ed6cba1c172b#diff-70fac99e0a455f1109c7e0ac5728bbc341ed445234d51ebf741850267a1b7f89L38

Reply:
It would be nice if we updated one of the multi-node tests to use basic auth to exercise this.  

We did exactly this in `PullQueryRoutingFunctionalTest` (though that also has an auth plugin installed I believe to use `authentication.skip.paths`, though this isn't a limitation anymore.  Though that test is disabled anyway now!

It would be good if we could basic auth to another IT that is multi node to try to ensure that we don't regress.  At the moment, I don't think there exists any such test.

Reply:
Added basic auth to `ShowQueriesMultiNodeFunctionalTest` to ensure no regression